### ISSUE TITLE
Play doesn't wait for request to be terminated before stop (TERM signal)

### ISSUE THREAD

**Issue by mickaelmagniez** [playframework/playframework#9737]
### Play Version
2.7.3

### API
Scala

### Operating System
Linux laptop 5.0.0-29-generic #31-Ubuntu SMP Thu Sep 12 13:05:32 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux

### JDK
openjdk version "11.0.4" 2019-07-16

### Expected Behavior
When sending TERM to play app, current request should finish

### Actual Behavior
When sending TERM to play app, current request doesn't finish

### Reproducible Test Case
Create new app from seed
```
sbt new playframework/play-scala-seed.g8
```
Replace HomeController::index by
```
 def index() = Action { implicit request: Request[AnyContent] =>
    println("Before sleep")
    Thread.sleep(10000)
    println("After sleep")
    Ok(views.html.index())
  }
```
Run in different terminals
```
sbt dist
unzip target/universal/play-scala-seed-1.0-SNAPSHOT.zip
play-scala-seed-1.0-SNAPSHOT/bin/play-scala-seed
curl http://localhost:9000
ps aux |grep "play-scala-seed-1.0-SNAPSHOT"  | awk '{print $2}' | xargs kill -s TERM
```

Logs are 
```
[info] p.c.s.AkkaHttpServer - Listening for HTTP on /0:0:0:0:0:0:0:0:9002
Before sleep
[info] p.c.s.AkkaHttpServer - Stopping server...
```

**Comment by ignasi35** [comment]
A `SIGTERM` triggers the execution of the Play's shutdown mechanism (which is implemented using Akka`s Coordinated Shutdown). The shutdown mechanism uses some default timeouts which can be tuned/relaxed but in `2.7.x` Play already uses Akka-HTTP's `unbind` which stops accepting requests but should allow in-flight requests to complete.

I see in you reproducer that the code in the `Action` sleeps for 10 seconds which is the default timeout in Akka Coordinated Shutdown. So, can you reproduce the issue using a `sleep` of _only_ 3 seconds? 

I'm trying to determine if this is an issue or the expected behavior.

related: https://doc.akka.io/docs/akka-http/current/server-side/graceful-termination.html, https://doc.akka.io/docs/akka/current/actors.html#coordinated-shutdown and https://github.com/playframework/playframework/blob/2.7.x/transport/server/play-akka-http-server/src/main/scala/play/core/server/AkkaHttpServer.scala#L492

**Comment by mickaelmagniez** [comment]
Same problem with 3 seconds, and clearly application stops immediately after sending SIGTERM, no delay

**Comment by dwijnand** [comment]
Fixed in #9970.
